decidim-core/app/cells/decidim/user_activity/show.erb

```diff
@@ -1,32 +1,24 @@
-<div class="user-activity">
-  <div id="activities" class="columns small-12">
-    <% if activities.length == 0 %>
-      <div class="callout warning">
-        <%= t "decidim.user_activity.index.no_activities_warning" %>
-      </div>
-    <% else %>
-      <div class="row">
-        <div class="show-for-mediumlarge filters__left-legend">
-          <%= filter_form_for context[:filter] do |form| %>
-            <div class="columns mediumlarge-6 large-5">
-              <%= form.custom_select :resources,
-                                     :resource_type,
-                                     resource_types,
-                                     legend_title: t("title", scope:"decidim.components.component_order_selector"),
-                                     label: false,
-                                     selected: context[:filter].resource_type,
-                                     include_blank: t("all_types", scope: "decidim.components.component_order_selector.order") %>
-            </div>
-          <% end %>
-        </div>
-      </div>
-      <div class="row" id="activities-container">
-        <%= cell "decidim/activities", activities %>
-      </div>
-
-      <div class="row" id="decidim-paginate-container">
-        <%= decidim_paginate activities %>
-      </div>
+<div class="profile__activity">
+    <%= redesigned_filter_form_for context[:filter], url_for, class: "new_filter profile__activity__filter-container" do |form| %>
```

Comment:
The filters still have a vertical scroll on smaller screens:
![Filters vertical scroll](https://user-images.githubusercontent.com/864340/187222200-901f167e-3509-4167-ac85-9aefda5eeae5.png)

It also applies to 320px although I purposefully picked a narrow screen for the screenshot.

Reply:
Moved to #9670.